pkg/client/client_test.go

```diff
@@ -39,7 +39,15 @@ func TestMain(m *testing.M) {
 func generateURL(driver string) string {
 	switch driver {
 	case drivers.Postgres:
```

Comment:
Why postgresql const is not present here? Isn't it tested?

Reply:
Because the that constant is the one used at the time to test on CI. I can only use one at a time there.